cmd/minishift/cmd/logs.go

```diff
@@ -49,5 +50,6 @@ var logsCmd = &cobra.Command{
 
 func init() {
 	logsCmd.Flags().BoolVarP(&follow, "follow", "f", false, "Continously print the logs entries.")
+	logsCmd.Flags().Int64VarP(&tail, "tail", "t", -1, "Tail the logs entries till some number.")
```

Comment:
Prefer to have `Number of lines to show from the end of the logs` same as docker logs help section have, at the end that what we are executing.

```
$ docker logs -h
Flag shorthand -h has been deprecated, please use --help

Usage:	docker logs [OPTIONS] CONTAINER

Fetch the logs of a container

Options:
      --details        Show extra details provided to logs
  -f, --follow         Follow log output
      --help           Print usage
      --since string   Show logs since timestamp
      --tail string    Number of lines to show from the end of the logs (default "all")
  -t, --timestamps     Show timestamps
```

Reply:
Updating in short